src/content/docs/en/technology/security/audits-and-bug-bounty.mdx

```diff
@@ -52,9 +53,13 @@ Scroll has worked with several industry-leading security audit firms to review o
   - [USDC Gateway](https://blog.openzeppelin.com/scroll-usdc-gateway-audit)
   - [Contract diff](https://blog.openzeppelin.com/scroll-diff-audit-report)
   - [Bridge Gas Optimizations](https://blog.openzeppelin.com/scroll-bridge-gas-optimizations-audit)
+  - [EIP-4844 Support](https://blog.openzeppelin.com/scroll-eip-4844-support-audit)
+  - [Batch Token Bridge](https://blog.openzeppelin.com/scroll-batch-token-bridge-audit)
+  - 
```

Comment:
Need to drop this.

![image](https://github.com/scroll-tech/scroll-documentation/assets/4205837/d824a7f6-68f9-4907-ac11-0c5521ad21a8)